src/pages/WorldEditor/WorldEditorStartPanel.tsx

```diff
@@ -28,9 +28,11 @@ export const WorldEditorStartPanel: React.FC<WorldEditorStartPanelProps> = ({
   const isUpdatingWorld = isLoaded && worldId && world;
   const isCreatingWorld = isLoaded && !world && !worldId;
 
+  const sidebarClassName = !worldId ? "AdminDashboard__sidebar" : "";
+
   return (
     <AdminPanel>
-      <AdminSidebar>
+      <AdminSidebar className={sidebarClassName}>
```

Comment:
This can be resolved within WorldEditorStartPanel.scss without class names injection